code/modules/html_interface/nanotrasen/nanotrasen.dm

```diff
@@ -0,0 +1,175 @@
+/*
+Author: NullQuery
+Created on: 2014-09-25
+
+Extension to implement Nanotrasen styled windows.
+
+Additional procs:
+
+	hi.setEyeColor(color, client)
+
+Use this to set the color of the 'eye' in the top-left corner of the window.
+
+The client is optional and may be a /mob, /client or /html_interface_client object. It must be specified, since the eye icon is specific to a client.
+
+*/
+
+/datum/html_interface/nanotrasen/New()
+	. = ..()
+
+	// Add appropriate CSS and set the default layout.
+	src.head = src.head + "<link rel=\"stylesheet\" type=\"text/css\" href=\"nanotrasen.css\" />"
+	src.updateLayout("")
+
+/datum/html_interface/updateLayout(layout)
+	// Wrap the layout in our custom HTML
+	return ..("<div id=\"ntbgcenter\"></div><div id=\"content\">[layout]</div>")
+
+/datum/html_interface/specificRenderTitle(datum/html_interface_client/hclient, ignore_cache = FALSE)
+	// Update the title in our custom header (in addition to default functionality)
+	winset(hclient.client, "browser_\ref[src].uiTitle", list2params(list("text" = "[src.title]")))
+
+/datum/html_interface/nanotrasen/sendResources(client/client)
+	. = ..() // we need the default resources
+
+	client << browse_rsc('uiBg.png')
+	client << browse_rsc('uiBgcenter.png')
+	client << browse_rsc('nanotrasen.css')
+
+/datum/html_interface/nanotrasen/createWindow(datum/html_interface_client/hclient)
+	. = ..() // we want the default window
+
+	// Remove the titlebar
+	winset(hclient.client, "browser_\ref[src]", list2params(list(
+													"titlebar"          = "false"
```

Comment:
Consider indenting only 1 level in for list elements, this can easily get out of hand (eg this is 13 tabs in)